test.env

```diff
@@ -0,0 +1,9 @@
+DATABASE_URL="postgresql://test_postgres:test_postgres@localhost:5432/postgres"
```

Comment:
why do you need this in the repository?

Reply:
I use it in `schema.prisma` - [Link to docs](https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch/relational-databases/connect-your-database-typescript-postgresql)